templates/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,59 @@
-<!--
-1. [Read and sign the CLA](https://cla.js.foundation/webpack/webpack.js.org). This needs to be done only once. PRs that haven't signed it won't be accepted.
-2. Check out the [development guide](https://webpack.js.org/development/) for the API and development guidelines.
-3. Read through the PR diff carefully as sometimes this can reveal issues. The work will be reviewed, but this can save some effort.
--->
+### `Notable Changes`
+
+> ✏️ Describe the big picture of your changes here to communicate to the maintainers
+why we should accept this pull request. If it fixes a bug or resolves a feature
+request, be sure to link to that issue down below.
+
+#### `Commit Message Summary (CHANGELOG)`
+
+```
+commit message body...
+```
+
+### `Type`
+
+> ℹ️  What types of changes does your code introduce?
+
+> 👉 _Put an `x` in the boxes that apply and delete all others_
```

Comment:
I think checkboxes are unnecessary if we’re asking people to remove items.

Reply:
What do you suggest instead :) ?

Reply:
Either put Xs or remove items ;-)

Reply:
I would drop the [ ] bit and have only a list. The developer can then remove the bits that aren't needed.